docs/reference/images/index.md

```diff
@@ -48,3 +48,27 @@ All public images available in {{brand}} support the following image properties:
 * `os_version=${VERSION_ID}`: operating system version, such as `os_version=22.04`
 
 [Other properties](https://docs.openstack.org/glance/latest/admin/useful-image-properties.html) may also be set on individual images.
+For instance, the `image_original_user` property reflects the username of the default non-root user of the corresponding operating system -- and that piece of information may come in handy in various automation scenarios.
+So, to find out the username of the default non-root user in the `Ubuntu 22.04 Jammy Jellyfish x86_64` image, you can type the following:
+
+```console
+$ openstack image show -f json "Ubuntu 22.04 Jammy Jellyfish x86_64" \
```

Comment:
In case you have python (you usually have out of the box) and don't want to install third-party depndencies (like jq), you can also do it like that:

`python3 -c "print($(openstack image show 'Ubuntu 22.04 Jammy Jellyfish x86_64'  -c properties -f value)['image_original_user'])"`

Reply:
Yes, that works too. I am wondering, though, if the usage of `jq` "feels" more natural to most users -- at least compared to resorting to Python?